.github/workflows/release.yml

```diff
@@ -1,137 +1,104 @@
+        TITLE=$(parse-changelog CHANGELOG.md --title)
+        CHANGELOG=$(parse-changelog CHANGELOG.md $VERSION)
+        echo "version=$VERSION" >> "$GITHUB_OUTPUT"
+        echo "title=$TITLE" >> "$GITHUB_OUTPUT"
+        echo "changelog=$CHANGELOG" >> "$GITHUB_OUTPUT"
+
   create-release:
     runs-on: ubuntu-latest
-    outputs:
-      has-releases: ${{ steps.create-release.outputs.has-releases }}
     env:
       GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      RELEASE_TITLE: ${{ steps.metadata.outputs.title }}
+      RELEASE_CHANGELOG: ${{ steps.metadata.outputs.changelog }}
     steps:
-      - uses: actions/checkout@v3
-      - name: Install Rust
-        run: rustup update 1.68 --no-self-update && rustup default 1.68
-      - name: Install cargo-dist
-        run: curl --proto '=https' --tlsv1.2 -LsSf https://github.com/axodotdev/cargo-dist/releases/download/v0.0.5/cargo-dist-v0.0.5-installer.sh | sh
-      - id: create-release
-        run: |
-          cargo dist manifest --tag=${{ github.ref_name }} --artifacts=all --no-local-paths --output-format=json > dist-manifest.json
-          echo "dist manifest ran successfully"
-          cat dist-manifest.json
-        
-          # Create the Github Release™ based on what cargo-dist thinks it should be
-          ANNOUNCEMENT_TITLE=$(cat dist-manifest.json | jq --raw-output ".announcement_title")
-          IS_PRERELEASE=$(cat dist-manifest.json | jq --raw-output ".announcement_is_prerelease")
-          cat dist-manifest.json | jq --raw-output ".announcement_github_body" > new_dist_announcement.md
-          gh release create ${{ github.ref_name }} --draft --prerelease="$IS_PRERELEASE" --title="$ANNOUNCEMENT_TITLE" --notes-file=new_dist_announcement.md
-          echo "created announcement!"
-
-          # Upload the manifest to the Github Release™
-          gh release upload ${{ github.ref_name }} dist-manifest.json
-          echo "uploaded manifest!"
+      - run: |
+        echo "$RELEASE_CHANGELOG" >> release_changelog.md
+        gh release create ${{ github.ref_name }} --draft --title="$RELEASE_TITLE" --notes-file=release_changelog.md
+        gh release upload ${{ github.ref_name }} LICENSE CHANGELOG.md README.md
 
-          # Disable all the upload-artifacts tasks if we have no actual releases
-          HAS_RELEASES=$(cat dist-manifest.json | jq --raw-output ".releases != null")
-          echo "has-releases=$HAS_RELEASES" >> "$GITHUB_OUTPUT"
-
-  # Build and packages all the things
-  upload-artifacts:
-    # Let the initial task tell us to not run (currently very blunt)
-    needs: create-release
-    if: ${{ needs.create-release.outputs.has-releases == 'true' }}
+  build:
     strategy:
       matrix:
-        # For these target platforms
+        os: ['windows-latest', 'ubuntu-latest', 'macos-latest']
         include:
-        - os: macos-11
-          dist-args: --artifacts=local --target=aarch64-apple-darwin --target=x86_64-apple-darwin
```

Comment:
Would it be a lot of work to re-enable the M1 mac target? I know we've had requests for this on fossa-cli

Reply:
actually probably not, you're right

Reply:
Changed that over, will merge soon